<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="horizontalSpacing">
    <number>10</number>
   </property>
   <property name="verticalSpacing">
    <number>5</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Hb</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Df</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Hf</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="LineEdit" name="lineedit_df">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Db</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="LineEdit" name="lineedit_db">
     <property name="text">
      <string>.5</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="6">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushbutton_close">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushbutton_apply">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Hc</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="LineEdit" name="lineedit_hc">
     <property name="text">
      <string>.5</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="LineEdit" name="lineedit_hb">
     <property name="text">
      <string>1.5</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="LineEdit" name="lineedit_hf">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3" rowspan="10" colspan="3">
    <widget class="QLabel" name="label_image">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>m</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>m</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>m</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>m</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>m</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QGroupBox" name="groupbox_to">
     <property name="title">
      <string>Top Outlet</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="1">
       <widget class="LineEdit" name="lineedit_dto">
        <property name="text">
         <string>.2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LineEdit" name="lineedit_hto">
        <property name="text">
         <string>.2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Dto</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Hto</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>m</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>m</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QGroupBox" name="groupbox_so">
     <property name="title">
      <string>Side Outlet</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Dso</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Hso</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LineEdit" name="lineedit_hso">
        <property name="text">
         <string>.2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LineEdit" name="lineedit_dso">
        <property name="text">
         <string>.2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Eso</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="LineEdit" name="lineedit_eso">
        <property name="text">
         <string>.5</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>m</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>m</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>m</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QGroupBox" name="groupbox_spouted">
     <property name="title">
      <string>Spouted</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Ds</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>As</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LineEdit" name="lineedit_ds">
        <property name="text">
         <string>.2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LineEdit" name="lineedit_as">
        <property name="text">
         <string>60</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>m</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>°</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>mfixgui.widgets.base</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineedit_db</tabstop>
  <tabstop>lineedit_hb</tabstop>
  <tabstop>lineedit_df</tabstop>
  <tabstop>lineedit_hf</tabstop>
  <tabstop>lineedit_hc</tabstop>
  <tabstop>groupbox_to</tabstop>
  <tabstop>lineedit_dto</tabstop>
  <tabstop>lineedit_hto</tabstop>
  <tabstop>groupbox_so</tabstop>
  <tabstop>lineedit_dso</tabstop>
  <tabstop>lineedit_hso</tabstop>
  <tabstop>lineedit_eso</tabstop>
  <tabstop>groupbox_spouted</tabstop>
  <tabstop>lineedit_ds</tabstop>
  <tabstop>lineedit_as</tabstop>
  <tabstop>pushbutton_close</tabstop>
  <tabstop>pushbutton_apply</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
